frontend/src/assets/locales/ro/translation.json

```diff
@@ -13,5 +13,17 @@
       "dashboard": "Dashboard"
     },
     "collapse": "Restrange meniul"
+  },
+  "organization_profile": {
+    "title": "Profil organizație",
+    "button_label": "Editează",
+    "data_info": "Datele sunt preluate din ONGHub.",
+    "logo": "Logo organizație",
+    "name": "Denumire organizație",
+    "email": "Email",
```

Comment:
email, phone, address are generic namings not specific to the organization and should be present in "general", please refer to the PR above.